### ISSUE TITLE
`cy.screenshot()` times out in Electron

### ISSUE THREAD

**Issue by MikeMcC399** [cypress-io/cypress#30172]
### Current behavior

`cy.screenshot()` times outs on Ubuntu 24.04 LTS with Electron browser when a spec is run from the CLI.

The issue does not occur with Chrome or Firefox and it is a regression compared to Cypress `13.13.3`.

### Desired behavior

`cy.screenshot()` should capture a screenshot and not time out.

### Test code to reproduce

Scaffold Cypress with single spec. Add `cy.screenshot()`.

```js
describe('template spec', () => {
  it('passes', () => {
    cy.visit('https://example.cypress.io')
    cy.screenshot()
  })
})
```

Execute the following in a Ubuntu terminal window (not VSCode):

```shell
npx cypress run
```

Running the same test spec in headed mode or using the Cypress Launchpad / App works correctly and does not time out.

```shell
npx cypress run --headed
npx cypress open
```

### Cypress Version

`13.14.1`

Regression from `13.13.3`

### Node version

`v20.17.0` LTS

### Operating System

Ubuntu `24.04.1` LTS

### Debug Logs

```text
1) template spec
       passes:
     CypressError: `cy.screenshot()` timed out waiting `30000ms` to complete.
      at https://example.cypress.io/__cypress/runner/cypress_runner.js:134531:73
      at tryCatcher (https://example.cypress.io/__cypress/runner/cypress_runner.js:1807:23)
      at https://example.cypress.io/__cypress/runner/cypress_runner.js:4186:41
      at tryCatcher (https://example.cypress.io/__cypress/runner/cypress_runner.js:1807:23)
      at Promise._settlePromiseFromHandler (https://example.cypress.io/__cypress/runner/cypress_runner.js:1519:31)
      at Promise._settlePromise (https://example.cypress.io/__cypress/runner/cypress_runner.js:1576:18)
      at Promise._settlePromise0 (https://example.cypress.io/__cypress/runner/cypress_runner.js:1621:10)
      at Promise._settlePromises (https://example.cypress.io/__cypress/runner/cypress_runner.js:1697:18)
      at _drainQueueStep (https://example.cypress.io/__cypress/runner/cypress_runner.js:2407:12)
      at _drainQueue (https://example.cypress.io/__cypress/runner/cypress_runner.js:2400:9)
      at Async._drainQueues (https://example.cypress.io/__cypress/runner/cypress_runner.js:2416:5)
      at Async.drainQueues (https://example.cypress.io/__cypress/runner/cypress_runner.js:2286:14)
```

**Comment by MikeMcC399** [comment]
I was unable to reproduce this issue in the beta GitHub runner image [Ubuntu 24.04 LTS](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2404-Readme.md).

**Comment by jennifer-shehane** [comment]
Ah, really thought we fixed all of these instances.

**Comment by jennifer-shehane** [comment]
Can't see why this would start happening in 13.14.0

**Comment by MikeMcC399** [comment]
@jennifer-shehane 

- The problem is reproducible on multiple machines, but not everywhere, however I don't know what the common factor is, apart from it being Ubuntu `24.04.x`.
- This is possibly related to the other regression in Cypress `13.14.0` & `13.14.1` https://github.com/cypress-io/cypress/issues/30160 on Ubuntu `24.04.x`.

**Comment by jennifer-shehane** [comment]
@MikeMcC399 Something that could help is narrowing down the commit this started from in 13.14.0. So ensure you can recreate the issue while running Cypress locally via the repo, then run `git bisect` narrowing down the commits between 13.13.1 release and 13.14.0 to see which commit the repro fails in and which it does not.

**Comment by MikeMcC399** [comment]
@jennifer-shehane 

- I was unable to reproduce starting `yarn cypress:run --project ...` however I could pin it down to PR #30109 by using beta versions.
- If I install the beta software from #30073 https://cdn.cypress.io/beta/npm/13.14.0/linux-x64/develop-507be6ce1b8cc1ef891462dff7a81968b540dfe9/cypress.tgz everything works. Moving to the next commit from PR #30109 https://cdn.cypress.io/beta/npm/13.14.0/linux-x64/develop-50335a0d2bbcdb99babed7f8fb038132dede11b9/cypress.tgz introduces two regressions affecting Ubuntu `24.04.x`:
    - this issue #30172
    - #30160 

I suspect it may be necessary to revert this commit once again.

**Comment by AtofStryker** [comment]
> @jennifer-shehane
> 
> * I was unable to reproduce starting `yarn cypress:run --project ...` however I could pin it down to PR [misc: allow HiDPI Screen running wayland to use cypress window/browser #30109](https://github.com/cypress-io/cypress/pull/30109) by using beta versions.
> * If I install the beta software from [chore(deps): update babel monorepo #30073](https://github.com/cypress-io/cypress/pull/30073) https://cdn.cypress.io/beta/npm/13.14.0/linux-x64/develop-507be6ce1b8cc1ef891462dff7a81968b540dfe9/cypress.tgz everything works. Moving to the next commit from PR [misc: allow HiDPI Screen running wayland to use cypress window/browser #30109](https://github.com/cypress-io/cypress/pull/30109) https://cdn.cypress.io/beta/npm/13.14.0/linux-x64/develop-50335a0d2bbcdb99babed7f8fb038132dede11b9/cypress.tgz introduces two regressions affecting Ubuntu `24.04.x`:
>   
>   * this issue [`cy.screenshot()` times out in Electron #30172](https://github.com/cypress-io/cypress/issues/30172)
>   * [GLib-GIO-ERROR: Cannot start cypress on Ubuntu 24 - Settings schema 'org.gnome.settings-daemon.plugins.xsettings' does not contain a key named 'antialiasing' #30160](https://github.com/cypress-io/cypress/issues/30160)
> 
> I suspect it may be necessary to revert this commit once again.

@MikeMcC399 I'm struggling to reproduce on docker Ubuntu 24.04, but I can reproduce on my Jammy Ubuntu 22 machine. I am unsure of the root cause, so we are going to back out #30109 and release `13.14.2` tomorrow EST.

**Comment by MikeMcC399** [comment]
@AtofStryker

> I'm struggling to reproduce on docker Ubuntu 24.04, but I can reproduce on my Jammy Ubuntu 22 machine. I am unsure of the root cause, so we are going to back out #30109 and release `13.14.2` tomorrow EST.

- I also had problems to reproduce the `cy.screenshot()` timeout error with Docker. I tried unsuccessfully to reproduce on `cypress/base-internal:ubuntu24-node18`.

- I first noticed that it was failing when I ran the suite of `cypress-example-kitchensink` tests as I was upgrading my Ubuntu environments, however due to the fact that I couldn't reproduce it in all environments I delayed opening this issue until I could confirm it could at least be reproduced on two independent systems.

- Backing out #30109 and releasing a new patch should solve this issue. 👍🏻
